### ISSUE TITLE
"meadow" as genus in PEP data

### ISSUE THREAD

**Issue by jdeck88** [jdeck88/pheno_paper#23]
PEP725 data contains "meadow" as a genus in the genus field.  I believe all of the meadow records
are associated with silage/hay cutting.
Thus, they would be an instance of leaf presence. ... just no acceptable scientific name (or genus).  Not sure how to handle these.

**Comment by ramonawalls** [comment]
Right. We don't have a genus for this. We could call it a meadow community and add a term in PCO (or use ENVO), but the best we could do taxonomically is probably Viridiplantae (green plants). 

Can you just leave scientific name blank and include only common name, or does that mess up your parsing?

**Comment by jdeck88** [comment]
We'll encounter alot more of these when we start working with phenocam data...   How do other projects categorize "green-up"?, which this is perhaps a slightly more sophisticated measure of?

**Comment by ramonawalls** [comment]
True, but I prefer to cross that bridge when we have funding to cross it, and just focus on species for now.
